tests/integration/accounts/test_contract_account.py

```diff
@@ -91,3 +93,27 @@ async def test_should_give_infinite_allowance_to_kakarot(
                 )
                 == "Uint256(low=340282366920938463463374607431768211455, high=340282366920938463463374607431768211455)"
             )
+
+        async def test_should_take_deployment_fees(
```

Comment:
forgot to say but this test is ill placed. This tests a kakarot entrypoint and should consequently be in the test_kakarot.py file, in a `TestDeployExternallyOwnedAccount` class

Reply:
Can you explain the difference between these two tests:
- `def test_should_deploy_starknet_contract_at_corresponding_address`
- `def test_should_deploy_starknet_contract_with_correct_implementation`

Reply:
I was thinking about the class hash, to make sure that it actually deploys an EOA. But let' not block for this